list/source/GridListImageItem.js

```diff
@@ -146,12 +154,22 @@
 				this.selectedChanged();
 				this.imageSizingChanged();
 				this.centeredChanged();
+				this.placeholderChanged();
 			};
 		}),
 
 		/**
 		* @private
 		*/
+		placeholderChanged: function(){
+			if(this.imageSizing){
+				this.placeholder= this.placeholder? this.placeholder : enyo.Image.placeholder; 
```

Comment:
Instead of doing this, can we just set the value of `placeholder` in `moon.GridListImageItem` to `enyo.Image.placeholder`? By default this placeholder image will be used, otherwise the developer can overwrite it with something else.